R/modules.R

```diff
@@ -449,7 +452,7 @@ print.teal_modules <- function(x, ...) {
 
 #' Prints `teal_module`
 #' @param x (`teal_module`)
-#' @param ... arguments passed to other methods.
+#' @param ... ignored
```

Comment:
Is it? You can pass `indent` here.

Reply:
print.teal_module uses print.teal_modules that uses format that is dispatched by the class so it uses format.teal_module that actually does not use `...` at all

```r
#' @inheritParams format.teal_modules
#' @param x (`teal_module`)
#' @export
#' @rdname module
format.teal_module <- function(x, indent = 0, ...) { # nolint
  paste0(paste(rep(" ", indent), collapse = ""), "+ ", x$label, collapse = "")
}
```

Reply:
oh, I'm idiot. I was paying attention to `...` not being used in format.

Reply:
ok this should be either `arguments passed to other methods.` or `for print(): arguments passed to other methods and for format(): ignored`

Reply:
great, me too. reverting this

Reply:
178d922b899085b787df07d4e8ff3e159b9a7be8